docs/gherkins/ModeratingProposals.feature

```diff
@@ -0,0 +1,31 @@
+Feature: While using Modernator it becomes important to indicate which
+  Proposals have been moderated. This indicates that Proposers should focus
+  their attention on unmoderated and new Proposals. The moderation pattern is
+  one that is suitable for a number of different tasks, below are some
+  examples:
```

Comment:
It's a mandatory functionality for complete implementation, but it's not mandatory for moderators to use that functionality. In the lunch example the ranking might be enough and there might be no need to moderate anything.

Additionally, I think in the upvote description it's specified that only unmoderated things can be upvoted.